packages/astro/src/content/data-store.ts

```diff
@@ -34,8 +34,6 @@ export interface DataEntry<TData extends Record<string, unknown> = Record<string
 	 */
 	deferredRender?: boolean;
 	assetImports?: Array<string>;
-	/** @deprecated */
-	legacyId?: string;
```

Comment:
Internal API AFAIK so I didn't document the change. cc @ascorbic

Reply:
Yeah, that's fine. This should've been removed as part of removing the backwards-compat. It was deprecated as soon as it was added.